freeipa/server/common.sls

```diff
@@ -25,13 +25,15 @@ freeipa_server_pkgs:
 ldap_secure_binds:
   cmd.run:
     - name: |
```

Comment:
you may use: `output_loglevel: quiet`, but ENV looks better, as you still get the output and log's are not full of sensitive passwords.

the person who executes the command may any time get the pillar data and get the clear text password.

I agree that this doesn't have a better solution, unless: https://github.com/saltstack/salt/issues/26236 comes with kind of obfuscate solution.

Reply:
You need to also remember that this is a ONE TIME password. As soon as it is used, it is no longer valuable or sensitive. For it to be useful, you would need to have A) used the same OTP on multiple hosts or B) successfully snarf this from salt and then beat the FreeIPA installer to registration between OTP generation in salt and the host creation and the free-ipa-install registration or fail a run and take the place of that host.